projects/reticulum/lib/ret_web/controllers/auth_controller.ex

```diff
@@ -6,31 +6,49 @@ defmodule RetWeb.AuthController do
   alias Ret.User
   alias Ret.Repo
 
-  def request(_params) do
+  alias RetWeb.Router.Helpers
+
+  def request(conn, params) do
```

Comment:
probably should be `_,_` or `_conn,_params` since they are unused and otherwise elixir compiler will emit a warning